test/runnable/template9.d

```diff
@@ -4706,7 +4706,7 @@ void test14735()
 {
     char[64] buf;
 
-    // Supported from 2.063: (http://dlang.org/changelog#implicitarraycast)
+    // Supported from 2.063: (https://dlang.org/changelog#implicitarraycast)
```

Comment:
I think the intention to bookmark this is not successful, perhaps change it to point to `2.063` changelog specifically? It seems the changelog page changed to only show a table of content.